File: app/src/main/java/com/my/televip/virtuals/messenger/browser/Browser.java
package com.my.televip.virtuals.messenger.browser;

import android.content.Context;

import com.my.televip.Class.ClassLoad;
import com.my.televip.Class.ClassNames;
import com.my.televip.obfuscate.ArgsResolver;
import com.my.televip.obfuscate.Obfuscate;

import de.robv.android.xposed.XposedHelpers;

public class Browser {

    public static void openUrl(Context context, String url){
        XposedHelpers.callStaticMethod(
                ClassLoad.getClass(ClassNames.BROWSER),
                Obfuscate.getMethodName("Browser", "openUrl"), context, url
        );
    }

}

File: app/src/main/java/com/my/televip/virtuals/messenger/browser/Browser.kt
package com.my.televip.virtuals.messenger.browser

import android.content.Context
import com.my.televip.clazz.ClassLoad
import com.my.televip.clazz.ClassNames
import com.my.televip.obfuscate.Obfuscate
import de.robv.android.xposed.XposedHelpers

object Browser {

    @JvmStatic
    fun openUrl(context: Context?, url: String?) {
        val browserClass = ClassLoad.getClass(ClassNames.BROWSER) ?: return
        XposedHelpers.callStaticMethod(browserClass, Obfuscate.getMethodName("Browser", "openUrl"), context, url)
    }
}
